### PR TITLE
[SYN5-718]

### PR DESCRIPTION
**Pull request by mgebala8**

### CODE REVIEW

**Comment by mkucharz** [comment]
@mgebala8 as far as I know @Idered told you to revert the previous merge and then create PR once again? This way we are starting to have a big mess.

File: src/components/ui/sign-description.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
 const SignDescription = () => (
   <div>
```
Comment (Idered) [inline src/components/ui/sign-description.js:2]: `className='SignDescription'`

File: src/components/ui/sign-description.js [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
-    <div className="imageWrapper">
+    <div className="img--wrap">
```
Comment (Idered) [inline src/components/ui/sign-description.js:3]: `className='SignDescription__img'`

File: src/components/ui/sign-description.js [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
       <img src="https://media.licdn.com/mpr/mpr/shrinknp_200_200/p/5/005/084/138/2e66790.jpg" />
     </div>
     <span>We managed to build, launch and succeed with our Pokemon Go map app in just 48 hours of programming.
-    <author>Artur Czemiel<span>developer & founder, Aexol</span></author>
+    <span className="author">Artur Czemiel<span>developer & founder, Aexol</span></span>
```
Comment (Idered) [inline src/components/ui/sign-description.js:7]: `className='SignDescription__author'`

File: src/components/ui/sign-description.js [hunk 4]
```diff
@@ -17,46 +17,45 @@ const SignDescription = () => (
...
+        opacity:0.8;
+        font-family:Roboto;
+        font-size:16px;
+        line-height:1.12;
+        color:#3e434a;
         margin-top:20px;
       }
 
-      author span{
+      .author span{
```
Comment (Idered) [inline src/components/ui/sign-description.js:49]: Does this selector match anything?

File: src/components/ui/sign-description.js [hunk 5]
```diff
@@ -17,46 +17,45 @@ const SignDescription = () => (
...
         display:block;
         text-align:left;
-  	    width: auto;
-  	    height: 18px;
-      	opacity: 0.8;
-      	font-family: Roboto;
-      	font-size: 16px;
-      	line-height: 1.12;
-      	color: #3e434a;
+        width:auto;
```
Comment (Idered) [inline src/components/ui/sign-description.js:39]: Are those properties copy paste from zeplin? You don't need some of them.

**Pull request closed without merging**